.golangci.yml

```diff
@@ -14,10 +14,16 @@ linters:
     - exhaustive
     - makezero
     - nakedret
+    - copyloopvar
     # - goconst # TODO: enable and fix issues
   fast: false
 
 linters-settings:
+  copyloopvar:
+    # Check all assigning the loop variable to another variable.
+    # Default: false
+    # If true, an assignment like `a := x` will be detected as an error.
+    check-alias: true
```

Comment:
If `check-alias` is `false`, an assignment to a variable with another name will not be detected an error.

```go
for _, i := range someSlice {
    x := i
}

```

I think `check-alias` should be `true`.

Reply:
I agree.